processors/piifilterprocessor/filters/regexmatcher/regex.go

```diff
@@ -70,7 +54,7 @@ func (rm *Matcher) FilterStringValueRegexs(value string, key string, path string
 	return filtered, value
 }
 
-func (rm *Matcher) replacingRegex(value string, key string, regex *regexp.Regexp, redactor redaction.Redactor) (bool, string) {
+func (rm *Matcher) replacingRegex(value string, _ string, regex *regexp.Regexp, redactor redaction.Redactor) (bool, string) {
```

Comment:
nit: was there a reason to not use remove this arg?

Reply:
Since that was unrelated to the change but lint was complaining I decided to keep it but it is also OK to remove it.